### PR TITLE
Portfolio set up (Mobile first navbar and headline section)

### PR DESCRIPTION
**Pull request by Bettenoch**
Made the following changes:
Added navbar section with logo and harmburger menu. 
Added Headline section.
Added  linters with lighthouse, Weblint and stylint properties.

Thank you for reviewing my repository

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                <li> <a href="#" class="active">About</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+            <div class="menu_icons">
+                <a href="#"><img class="harmburgericon" src="/Assets/images/Harmburger Menu.svg" alt="Closeicon"></a>
+                <a href="#"><img class="closeicon" src="/Assets/images/Closeicon.svg" alt="Closeicon"></a>
+            </div>
+        </nav>
+    </header>
```
Comment (V-Blaze) [inline index.html]: - [ ] You did a great job using the `header and nav` semantic tag here. :+1: :100: but looking at the screenshot below, you can see that this section has some styling issues,
- [ ] The logo and the hamburger menu icon images are not displaying, I think this is due to the `src` URL. The first folder should be `assets` as you named it and not `Assets`, these file names are case sensitive. I'll suggest you update this accordingly. :smile: :small_airplane: 

- - Screenshot:
Screenshot from 2023-06-07 13-50-41

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                            <li><a href="#"><img src="/Assets/images/github.svg" alt="github"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/Linkedin icon.svg" alt="Linkedin"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/angel list.svg" alt="angellist"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/medium.svg" alt="Medium"></a> </li>
+                        </ul>
+                    </div>
+                    <img class="background-image" src="/Assets/images/Illustration.svg" alt="header-banner">
+                </div>
+
+
```
Comment (V-Blaze) [inline index.html]: - [ ] You did great using a section semantic tag here also, :+1: :+1: 
- [ ] Just like I pointed out above, you can see that your images are not displaying properly, I'll suggest you update their src URL. :+1: 
- [ ] The According to the Figma design, the background color is not white, I'll suggest you update this accordingly. :+1: 
- [ ] As you can see from the video below, the paragraph text is not visible, I think you need to check the styling for this section and make sure you are using style rules supported by all browsers.

- - YOUR PROJECT VIDEO:- 

https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/107632780/45d43120-d0ed-44d9-80a3-6ea34bc03b38

_For reference this project was opened in Chrome browser_

- [ ] FIGMA DESIGN:
Screenshot from 2023-06-07 13-55-55

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Bettenoch  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Well-structured code✔
-  Descriptive pull request summary👋 
-  descriptive pull request title✔

 ## Required Changes ♻️
- [ ] I'll suggest you remove these two placeholder comments from your readme file, there were added there to guide you and were meant to be removed while customizing your readme
- - Screenshot:
Screenshot from 2023-06-07 13-59-40

- - Screenshot from 2023-06-07 14-00-01

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bettenoch** [comment]
@V-Blaze Thank You for reviewing mg code. I have made the changes as requested how should I Proceed?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            <ul class="nav-list">
+                <li> <a href="#" class="active">About</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+            <div class="menu_icons">
+                <a href="#"><img class="harmburgericon" src="/Assets/images/Harmburger Menu.svg" alt="Closeicon"></a>
+                <a href="#"><img class="closeicon" src="/Assets/images/Closeicon.svg" alt="Closeicon"></a>
+            </div>
+        </nav>
```
Comment (Ibizugbe) [inline index.html]: Nice job fixing the broken links as requested by the previous reviewer👍. Currently, your toolbar has a lot of spacings (top and bottom) making it look a bit different from the given design😉. I think you can make the image logo smaller so as to retain the height of the toolbar as shown in the design😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

File: index.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                    
+                    <div class="socialmedia">
+                        <ul class="socialmedia-icons">
+                            <li><a href="#"><img src="/Assets/images/github.svg" alt="github"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/Linkedin icon.svg" alt="Linkedin"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/angel list.svg" alt="angellist"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/medium.svg" alt="Medium"></a> </li>
+                        </ul>
+                    </div>
+                    <img class="background-image" src="/Assets/images/Illustration.svg" alt="header-banner">
```
Comment (Ibizugbe) [inline index.html:69]: I suggest you work on the position of the background image so as to match the given design😉. currently, as a result of the current position of the background image, some of the texts are hard to see which can lead to a frustrating user experience😉. kindly ensure your background image is rightly positioned as shown in the design😉. try to consider the spacings around the image in the design👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Bettenoch 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍.
- [x] Broken links to images have been fixed as requested by the previous reviewer👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bettenoch** [comment]
Hello @V-Blaze I have made changes to my project  but I constantly getting this Weblint error regarding my node. Someone has advised that I remove the Weblint section on linters.yml file from line 24 to 29 but I know that we should confirm from our code reviewers. Please advise on how I should proceed.
node error

**Comment by Bettenoch** [comment]
Hello @Ibizugbe  I have made changes to my project but I am constantly getting this Weblint error regarding my node. Someone has advised that I remove the Weblint section on linters.yml file from line 24 to 29 but I know that we should confirm from our code reviewers. Please advise on how I should proceed.
node error

**Comment by Ibizugbe** [comment]
Hello @Bettenoch 👋👋, There's an allowance for this error today😉. You can proceed to submit, and your project should be reviewed without any considerations to the error👍. 

But on your own, I suggest you try reinstalling linters on the project. take all the linter files from your local computer and from github (the branch with this PR and the main branch). after that, download the latest linters from the repo and setup everything afresh😉. I think this might work😉.

**Comment by Bettenoch** [comment]
> 

Okay Thank You @Ibizugbe  I will consider reinstalling Linter files once again.

**Comment by Ibizugbe** [comment]
Hello @Bettenoch , I didn't suggest you submit a project without webhint. There's an allowance for that webhint error and a reviewer can only proceed to review your project if that error is seen. I suggest you kindly add the webhint file and only submit if that error persists👍.

**Review by IngDavidTM — APPROVED** [review]
Hi @Bettenoch 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [OPTIONAL] You did an outstanding job implementing the necessary changes :+1:. However, I've noticed that in your HTML and CSS files, the folder for assets is referred to as "Assets" instead of "assets." This inconsistency is causing issues with the images. I kindly request you to correct it :+1:.

This is how it looks on my device (I noticed that you have the same issue in the previous request that's why I request it as Optional, but it is important to change the directory:

Screenshot from 2023-06-10 06-27-42



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**